datalab/mlalpha/_cloud_predictor.py

```diff
@@ -32,29 +32,22 @@
 class CloudPredictor(object):
   """Preforms cloud predictions on given data."""
 
-  def __init__(self, model_name, version_name, metadata_path=None, label_output=None,
+  def __init__(self, model_name, version_name, label_output=None, 
```

Comment:
We may need to remove it for both local LocalPredictor and CloudPredictor, but have not decided yet. It depends on whether the online prediction returns labels or integers in classification. It is  possible that we will still need it.

I will add a TODO here and probably today will send another change for this once we decided.